Cargo.toml

```diff
@@ -5,6 +5,7 @@ members = [
     "slatedb-bencher",
     "slatedb-cli",
     "slatedb-dst",
+    "slatedb-examples",
```

Comment:
Nit: WDYT about just calling this folder "examples" like you did for lambda? That feels a bit more idiomatic to me.

Reply:
updated.